sidecarDockerrun.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+while true; do
+    TOKEN_JSON=`curl  http://workspace-token-service.$NAMESPACE/token/ | jq -r '.token'`
+    sleep 1
+    MANIFESTEXT=`curl https://$HOSTNAME/manifests/ -H "Authorization: Bearer $TOKEN_JSON" | jq --raw-output .manifests[-1].filename`
+    if [ $MANIFESTEXT = 'null' ]; then
+        # user doens't have any manifest
+        continue
+    fi
+    FILENAME=`echo $MANIFESTEXT | sed 's/\.[^.]*$//'`
+    if [ ! -d /data/$FILENAME ]; then
+        curl https://$HOSTNAME/manifests/file/$MANIFESTEXT -H "Authorization: Bearer $TOKEN_JSON"  > ~/manifest.json
+        gen3fuse ~/fuse-config.yaml ~/manifest.json /data/$FILENAME https://$HOSTNAME http://workspace-token-service.$NAMESPACE
+    fi
+done
```

Comment:
I think for now - as a first cut - we probably want the sidecar to enter a do-nothing loop after it mounts a manifest rather than loop and check for more manifests from the manifest service?

Reply:
ok - cool - if we want to auto-merge new manifests as they appear, but add another `sleep 20` before `done` ...

Reply:
actually yeah, we might do that until we figure out a limit of number of manifests we allow mounting for the users.